.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+  "tailwindCSS.experimental.classRegex": [
```

設定の背景が伝わるように補足のコメントを追加してください。

```suggestion
  // https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659
  "tailwindCSS.experimental.classRegex": [
```

---

https://github.com/ricora/order/pull/98/commits/78c972bc73513cfcefa0871843fac8a64d2e5ff4 にて対応しました